ios/engine/KMEI/KeymanEngine/Classes/Log.swift

```diff
@@ -8,4 +8,19 @@
 
 import XCGLogger
 
-public let log = XCGLogger(identifier: "KeymanEngine", includeDefaultDestinations: true)
+public let log: XCGLogger = {
+  // Default:  the 'console', which is read by Xcode but doesn't reach the system logs.
+  let mainLog = XCGLogger(identifier: "KeymanEngine", includeDefaultDestinations: true)
+
+  // Ensures our log messages go out to the device's system log.
+  let systemLogDest = AppleSystemLogDestination(identifier: "")
+  systemLogDest.showLogIdentifier = true
+#if DEBUG
+  systemLogDest.outputLevel = .debug
+#else
+  systemLogDest.outputLevel = .warning
+#endif
```

Comment:
We can control which log messages actually reach the system logs via this parameter.

I don't see anything that explicitly states the severity ordering offered by XCGLogger, but there's an implied ordering that can be found two paragraphs above this link's target:  https://github.com/DaveWoodCom/XCGLogger#advanced-usage-recommended.

Our iOS code relies _mostly_ on `log.debug`, `log.info` and `log.error`, with a few exceptions:

<img width="505" alt="Screen Shot 2021-02-22 at 12 15 32 PM" src="https://user-images.githubusercontent.com/25213402/108665516-aba6fa80-7507-11eb-933d-859602c7e360.png">

This pre-processor conditional ensures that `log.info` and `log.debug` messages won't reach the system logs for release versions of the framework/app.

Reply:
`log.error`, by comparison, has 100 results in 24 files, all of which indicate 'exceptional' circumstances we'd want to know about if a user were able to send us their logs, as they'd likely prove to be useful 'breadcrumbs' for diagnosing the problem.